From 70d8f7fb0694ddccd1ed3e8eb090955ca6e7f2a2 Mon Sep 17 00:00:00 2001
From: Philipp Trommler <philipp.trommler@mailbox.tu-dresden.de>
Date: Tue, 4 Oct 2016 09:27:57 +0200
Subject: [PATCH] Recreating `CMakeLists.txt` patch.

---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 89effcf..d1ea65a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,8 +6,6 @@ set(HASH_FUNCTION "HASH_OWN" CACHE STRING "Use other than identity function as h
 
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/common)
 
-find_package(Scorep REQUIRED)
-
 set(PLUGIN_SOURCE_FILES ${PROJECT_SOURCE_DIR}/src/dynamic-filtering.c)
 
 add_definitions("-Wall -Wextra -pedantic -std=c11 -DHASH_FUNCTION=${HASH_FUNCTION}")
@@ -16,5 +14,5 @@ if(BUILD_DEBUG)
 	add_definitions("-ggdb -DDYNAMIC_FILTERING_DEBUG")
 endif()
 
-include_directories(${PROJECT_SOURCE_DIR}/src/ ${SCOREP_INCLUDE_DIRS})
+include_directories(${PROJECT_SOURCE_DIR}/src/ ${PROJECT_SOURCE_DIR}/build/scorep_substrates_dynamic_filtering/include/ ${PROJECT_SOURCE_DIR}/build/scorep_substrates_dynamic_filtering/include/scorep/)
 add_library(${PROJECT_NAME} SHARED ${PLUGIN_SOURCE_FILES})
-- 
2.10.0

