From 01d935911bf220536613208fb62af065c54a9ab0 Mon Sep 17 00:00:00 2001
From: Philipp Trommler <philipp.trommler@mailbox.tu-dresden.de>
Date: Mon, 14 Nov 2016 17:37:49 +0100
Subject: [PATCH] Patch update.

---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ff23408..77bf512 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,8 +7,6 @@ set(MAX_THREAD_CNT "512" CACHE STRING "Change maximum number of observed threads
 
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/common)
 
-find_package(Scorep REQUIRED)
-
 set(PLUGIN_SOURCE_FILES ${PROJECT_SOURCE_DIR}/src/dynamic-filtering.c)
 
 add_definitions("-Wall -Wextra -pedantic -std=c11 -DHASH_FUNCTION=${HASH_FUNCTION} -DMAX_THREAD_CNT=${MAX_THREAD_CNT}")
@@ -17,5 +15,5 @@ if(BUILD_DEBUG)
 	add_definitions("-ggdb -DDYNAMIC_FILTERING_DEBUG")
 endif()
 
-include_directories(${PROJECT_SOURCE_DIR}/src/ ${SCOREP_INCLUDE_DIRS})
+include_directories(${PROJECT_SOURCE_DIR}/src/ ${PROJECT_SOURCE_DIR}/build/scorep_substrates_dynamic_filtering/include/ ${PROJECT_SOURCE_DIR}/build/scorep_substrates_dynamic_filtering/include/scorep/)
 add_library(${PROJECT_NAME} SHARED ${PLUGIN_SOURCE_FILES})
-- 
2.10.2

